metatron/src/main/java/uk/gov/ida/eidas/metatron/resources/MetatronResource.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.ida.eidas.metatron.resources;
+
+import uk.gov.ida.eidas.metatron.core.dto.EidasConfig;
+import uk.gov.ida.eidas.metatron.core.dto.EidasCountryConfig;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import java.security.KeyStore;
+import java.security.KeyStoreException;
+import java.util.Iterator;
+
+@Path("/")
+public class MetatronResource {
+
+    EidasConfig config;
+
+    public MetatronResource(EidasConfig config) {
+        this.config = config;
+    }
+
+    @GET
+    @Path("/config")
+    public String config() {
+        return config.getCountries().stream()
+                .map(EidasCountryConfig::toString)
+                .reduce("", (in, config) -> in + config) ;
+    }
+
+    @GET
+    @Path("/truststore")
+    public String truststore() throws KeyStoreException {
```

Comment:
This seems to be the big output - and although serving it as html is really helpful for us, would it be better served as JSON or ... something else machine readable?

Reply:
No, this was the original skeleton thing so it was just to show it was working.  Unfortunately though, as the PRs have taken so long, it's now behind expectations for what it's supposed to be doing now.

This was just to show humans that the truststore was created and ingested ...